packages/react-components/src/components/chart/types.ts

```diff
@@ -93,7 +93,7 @@ export type ChartOptions = {
   fontSettings?: SimpleFontSettings;
   legend?: ChartLegend;
   significantDigits?: number;
-  graphic?: InternalGraphicComponentGroupOption[];
+  graphic?: InternalGraphicComponentGroupOption[]; // This needs to be removed from the public api
```

Comment:
it should be renamed not removed, this is to allow users to add a TC by default

Reply:
When we were deciding on the API we had said this was not going to be an option? I thought this was only going to be an on the fly feature?

Reply:
The final API is obviously decide, but that the intention. not sure what fly feature means, but the intention was that if they want it, they can add.. we may or may not decide to keep this at the time of launch.